src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -114,18 +114,18 @@ If your existing application (`MyApp`) doesn't already have a Podfile, follow th
 
     <?code-excerpt "MyApp/Podfile" title?>
     ```ruby
-      flutter_application_path = '../my_flutter'
-      load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+    flutter_application_path = '../my_flutter'
+    load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
```

Comment:
@sfshaza2 I'm not sure if this will fix the problem.  Do you know?

Reply:
@jmagman, I tried staging your change, but wasn't able to. I am not sure why you are seeing this issue. I don't know if it's because it's marked as ruby code (and maybe that's broken), or whether it's because of the indentation on the line. If I could stage it, then I could play with it and tweak. I can stage branches of the main repo, but I can't seem to stage a branch off a forked repo.

Reply:
Would you like me to close this, then?  Sounds like it's not useful off a fork.  
I've seen GitHub issues where people are pasting the text in their code, including the misaligned whitespace.